DuckDuckGo/PrivacyProtectionOverviewController.swift

```diff
@@ -87,6 +87,22 @@ class PrivacyProtectionOverviewController: UITableViewController {
 
         return true
     }
+    
+    override func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        guard UIDevice.current.userInterfaceIdiom == .phone else { return }
+        
+        let frameHeight = tableView.frame.size.height
+        let contentHeight = tableView.contentSize.height
+        let height = frameHeight - contentHeight
+        let distance = height + scrollView.contentOffset.y
+        let percent = distance / scrollView.frame.size.height * 100
```

Comment:
Is there any danger of hitting a divide by zero here? I'm thinking of edge cases, e.g someone starts scrolling and then closes the viewcontroller by tapping on the grade button. If you think there's an edge-case risk, let's add a height zero check here.